text/dotnet-core/rfcs/0004-extend-dotnet-sdk.md

```diff
@@ -0,0 +1,50 @@
+# Extend .Net SDK Dependency
+
+## Proposal
+
+Reconfigure the .Net SDK dependency to be identical to the one that can be downloaded from Microsoft which includes the runtimes that the SDK is dependent on to run.
+
+## Motivation
+As of .Net SDK version ~6.0.100 publish now longer properly functions for certain types of .Net apps. Upon testing we were able to get the buildpack to function again if we used an .Net SDK dependency that still had the runtimes libraries packaged with it. After further investigation we discovered that the removal of those local runtimes was what was causing the failure, leading us to believe that the .Net SDK no longer follows the runtime search path that it used to.
+
+It is becoming ever more apparent to me that Microsoft is dead set on not having flexibility of file location during `dotnet publish` therefore I think that we should capitulate to Microsoft and create an expected .Net Hive installation. I think that this will make the buildpack less brittle to change in the future and will construct a build environment that is more similar to what .Net developers actually have installed on their machines and what Microsoft has in their Docker images.
```

Comment:
nit: can you link to .Net Hive docs you looked at?  It's a concept that is new to the buildpacks

Reply:
It would also be helpful if you described what a .Net Hive is, it's not clear that it includes runtime and ASP.Net

Reply:
That is fair, I don't know that it is an official term but I can put in an explanation.

Reply:
Could you take another look and see if I addressed your question. I gave a small expaination.